src/components/ResourceList/components/CheckableButton/CheckableButton.tsx

```diff
@@ -9,6 +14,7 @@ export interface CheckableButtonProps {
   label?: string;
   selected?: boolean | 'indeterminate';
   selectMode?: boolean;
+  smallScreen?: boolean;
```

Comment:
Should we create an issue to add `RL smallScreen` to media query provider and remove this prop?

Reply:
If you feel like we should we could do that. I actually don't mind the prop here making this a controllable dumb component.